velox/docs/develop/testing/spark-query-runner.rst

```diff
@@ -0,0 +1,75 @@
+==================
+Spark Query Runner
+==================
+
+Introduction
+------------
+
+The Spark Query Runner is a tool designed to facilitate the testing of Velox.
+It achieves this by executing SQL queries on Apache Spark and comparing the results
+with those produced by Velox. It helps ensure the correctness of Velox's computing
+against Spark and provides a method for identifying potential issues in Velox's
+implementation. Currently Spark-3.5.1 is used in Velox.
+
+How It Works
+------------
+
+The Spark Query Runner operates by executing given SQL queries on both Spark and Velox.
+The results from both systems are then compared to check for any differences.
```

Comment:
It seems like this description applies to Fuzzer, not Spark Query Runner.

Reply:
Thanks for the pointer. Moved to below part.

> Currently to use Spark as reference DB is only supported in aggregate fuzzer test, in which
the results from Velox and Spark are compared to check for any differences. If the results
match, it indicates that Velox is producing the correct output. If the results differ, it
suggests a potential issue in Velox that needs to be investigated. You can trigger its test
referring to :doc:`Fuzzer <fuzzer>`.